backend/api/models/user.py

```diff
@@ -16,8 +16,8 @@ class User(Base):
     name = Column(String)
     ae_title = Column(String)
     is_admin = Column(Boolean, default=False)
-    first_seen = Column(DateTime, default=datetime.utcnow)
-    last_seen = Column(DateTime, default=datetime.utcnow)
+    first_seen = Column(DateTime, default=datetime.now())
+    last_seen = Column(DateTime, default=datetime.now())
```

Comment:
@KlamChowder1 Is there any reason you changed this? Pretty this will just return the date the server was started as the default date. Other than that everything looks good.

```suggestion
    first_seen = Column(DateTime, default=datetime.now)
    last_seen = Column(DateTime, default=datetime.now)
```

Reply:
@awtkns yea the server would show the date but in UTC, so I just did `now() ` so that the datetime displayed would be in local time so users can read it more easily